workloads/network-perf/common.sh

```diff
@@ -95,11 +95,15 @@ export_defaults() {
       exit 1
     fi
   # If MULTI_AZ is disabled we use the two first nodes from the first AZ
-  else
-    nodes=($(oc get nodes -l node-role.kubernetes.io/worker,node-role.kubernetes.io/workload!="",node-role.kubernetes.io/infra!="",topology.kubernetes.io/zone=${zones[0]} -o jsonpath='{range .items[*]}{ .metadata.labels.kubernetes\.io/hostname}{"\n"}{end}'))
-    if [[ ${#nodes[@]} -lt 2 ]]; then
-      log "At least 2 worker nodes placed in the topology zone ${zones[0]} are required"
-      exit 1
+    else
```

Comment:
Same as w/ kubeburner. Why are there modifications to uperf when it is not involved in this pr seemingly?